cpanfile

```diff
@@ -103,6 +104,7 @@ feature 'debug', "Debug pane" =>
 # will be included, so put our testing requirements in develop...
 on 'develop' => sub {
     requires 'App::Prove', '3.36';
+    requires 'File::Find::Rule::Perl';
```

Comment:
@ylavoie where was this extra dependency introduced? I can't find it in the diff...

Reply:
Leftover. Removing